.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto
```

Comment:
Are all these 3 lines required ?

Reply:
I believe they are, because I'd prefer to make explicit how end of lines are to be handled for this project, as we have seen weirdness with line endings when cloning on windows.